src/v2/guide/single-file-components.md

```diff
@@ -1,48 +1,48 @@
 ---
-title: Single File Components (Componentes de un solo archivo)
+title: Componentes de un Solo Archivo (Single File Components)
 type: guide
 order: 402
 ---
 
 ## Introducción
 
-En muchos proyectos de Vue, los componentes globales serán definidos usando `Vue.component`, seguido de `new Vue({ el: '#container' })` para apuntar a un elemento contenedor en el body de cada página.
+En muchos proyectos de Vue los componentes globales son definidos usando `Vue.component`, seguido de `new Vue({ el: '#container' })` para apuntar a un elemento contenedor en el cuerpo de cada página.
 
-Esto puede funcionar muy bien para proyectos de tamaño pequeño a mediano, donde JavaScript es usado solamente para mejorar algunas vistas. Sin embargo, en proyectos con mayor complejidad, or donde tu front-end es enteramente basado conducido por JavaScript, estas desventajas se hacen evidentes:
+Esto puede funcionar muy bien para proyectos pequeños y medianos, donde JavaScript solo se usa para mejorar ciertas vistas. Sin embargo, en proyectos más complejos, o cuando la interfaz está exclusivamente impulsada por JavaScript, varias desventajas se hacen evidentes:
 
-- **Definiciones globales** fuerza a nombres unicos para cada componente
-- **Plantilla de texto** carecen de resaltado de sintaxis y requieren el uso de barras para HTML multilínea
-- **Sin soporte CSS** significa que mientras el HTML y JavaScript son modularizados en componentes, el CSS es omitido
-- **Sin compilación** nos restringe a HTML y ES5 JavaScript, en lugar de preprocesadores como Pug (anteriormente Jade) y Babel
+- Las **definiciones globales** nos obligan a usar nombres únicos para cada componente
+- Las **plantillas de texto** carecen de resaltado de sintaxis y requieren feas barras para generar HTML multilínea
+- La **ausencia de soporte para CSS** significa que, en tanto HTML y JavaScript son modularizados en componentes, el CSS queda visiblemente fuera
+- **Sin una etapa de compilación** nos vemos restringidos a HTML y JavaScript ES5, en lugar de preprocesadores como Pug (anteriormente Jade) y Babel
```

Comment:
- Maybe we can remove `JavaScript`. I think that if you are reading this part of the docs you should already know what ES5, ES6, etc. are, what do you think?
- Also, what do you think about adding a `-` in `preprocesadores`?
cc: @FLarra @sxcamacho 

```suggestion
- **Sin una etapa de compilación** nos vemos restringidos a HTML y ES5, en lugar de pre-procesadores como Pug (anteriormente Jade) y Babel
```

Reply:
Done